src/api/eth/call.js

```diff
@@ -0,0 +1,13 @@
+module.exports = (efx, wrapperAbi, address, action, args) => {
+  const { web3 } = efx
+
+  return new Promise((resolve, reject) => {
+    const contract = new web3.eth.Contract(wrapperAbi).at(address)
```

Comment:
@plutoegg this is where the [tests are failing](https://travis-ci.org/hems/efx-api-node/jobs/400148651#L654) should be an easy fix.... ( i guess )